### PR TITLE
Multiple code quality fix-1

### PR DESCRIPTION
**Pull request by faisal-hameed**
This pull request is focused on resolving occurrences of Sonar rules 
squid:ModifiersOrderCheck - Modifiers should be declared in the correct order.
squid:S1161 - @Override annotation should be used on any method overriding.
You can find more information about the issue heres: 
https://dev.eclipse.org/sonar/rules/show/squid:ModifiersOrderCheck
https://dev.eclipse.org/sonar/rules/show/squid:S1161

Please let me know if you have any questions.

Faisal Hameed

### CODE REVIEW

File: src/main/java/org/rassee/omniture/pig/AbstractSchemaGenerator.java [hunk 1]
```diff
@@ -39,8 +39,8 @@
 public abstract class AbstractSchemaGenerator implements SchemaGenerator {
     //A simple Set based lookup that simply defines columns for each data types
     //It is important to use right data type if we know for sure. This offers efficient storage of output in parquet or orc format
-    private final static Set<String> timestampColumns = Sets.newHashSet("date_time");
-    private final static Set<String> longColumns = Sets.newHashSet("browser_height", "browser_width", "search_engine",
+    private static final Set<String> timestampColumns = Sets.newHashSet("date_time");
+    private static final Set<String> longColumns = Sets.newHashSet("browser_height", "browser_width", "search_engine",
```
Comment (msukmanowsky) [inline src/main/java/org/rassee/omniture/pig/AbstractSchemaGenerator.java]: Does the ordering of these actually matter or is this just a style thing?

File: src/main/java/org/rassee/omniture/pig/DefaultSchemaGenerator.java [hunk 2]
```diff
@@ -30,7 +30,7 @@
...
...alfbstories\tsocialfbstorytellers\tsocialinteractioncount\tsocialinteractiontype\tsociallanguage\tsociallatlong\tsociallikeadds\tsocialmentions\tsocialowneddefinitioninsighttype\tsocialowneddefinitioninsightvalue\tsocialowneddefinitionmetric\tsocialowneddefinitionpropertyvspost\tsocialownedpostids\tsocialownedpropertyid\tsocialownedpropertyname\tsocialownedpropertypropertyvsapp\tsocialpageviews\tsocialpostviews\tsocialpubcomments\tsocialpubposts\tsocialpubrecommends\tsocialpubsubscribers\tsocialterm\tsocialtotalsentiment\tsourceid\tstate\tstats_server\tt_time_info\ttnt\ttnt_action\ttnt_post_vista\ttransactionid\ttruncated_hit\tua_color\tua_os\tua_pixels\tuser_agent\tuser_hash\tuser_server\tuserid\tusername\tva_closer_detail\tva_closer_id\tva_finder_detail\tva_finder_id\tva_instance_event\tva_new_engagement\tvideo\tvideoad\tvideoadinpod\tvideoadplayername\tvideoadpod\tvideochannel\tvideocontenttype\tvideopath\tvideoplayername\tvideosegment\tvisid_high\tvisid_low\tvisid_new\tvisid_timestamp\tvisid_type\tvisit_keywords\tvisit_num\tvisit_page_num\tvisit_referrer\tvisit_search_engine\tvisit_start_page_url\tvisit_start_pagename\tvisit_start_time_gmt\tweekly_visitor\tyearly_visitor\tzip";
```
Comment (msukmanowsky) [inline src/main/java/org/rassee/omniture/pig/DefaultSchemaGenerator.java]: Really hard to see what was actually changed in this diff. What did you modify?

Comment (faisal-hameed) [inline src/main/java/org/rassee/omniture/pig/DefaultSchemaGenerator.java]: @msukmanowsky I just changed ordering.
`private static final String`

File: src/main/java/org/rassee/omniture/pig/AbstractSchemaGenerator.java (hunk 1 above)
Comment (faisal-hameed) [inline src/main/java/org/rassee/omniture/pig/AbstractSchemaGenerator.java]: @msukmanowsky It is just java convention to use modifiers in this order. It does not effect any logic.

**Pull request merged**